dbt/include/snowflake/macros/materializations/seed.sql

```diff
@@ -38,9 +38,11 @@
 
 {% materialization seed, adapter='snowflake' %}
     {% set original_query_tag = set_query_tag() %}
+    {% set  grant_config = config.get('grants') %}
 
     {% set relations = materialization_seed_default() %}
 
+    {% do apply_grants(relation, grant_config, should_revoke=True) %}
```

Comment:
This will be handled within `materialization_seed_default`, so it's not needed here